src/compute-pr-actions.ts

```diff
@@ -122,6 +126,7 @@ interface ExtendedPrInfo extends PrInfo {
     readonly otherOwners: string[];
     readonly noOtherOwners: boolean;
     readonly tooManyOwners: boolean;
+    readonly editsOwners: boolean;
     readonly canBeMerged: boolean;

```

Comment:
I know this is out of scope, but it just came up in discord: this should probably be renamed to `canBeSelfMerged` or `autoMerged` / `canBeOwnerMerged` or something like that

Reply:
OK, I'll do that in the next PR.